src/components/posts/PostList.tsx

```diff
@@ -0,0 +1,68 @@
+import { Grid, makeStyles, Tooltip, Typography } from '@material-ui/core';
+import { Add } from '@material-ui/icons';
+import { FC } from 'react';
+import * as React from 'react';
+import { useDialog } from '../Dialog';
+import { Fab } from '../Fab';
+import { List } from '../List';
+import { CreatePost } from './CreatePost';
+import { PostableIdFragment } from './PostableId.generated';
+import { PostListItemCard } from './PostListItemCard';
+import { PostListItemCardFragment } from './PostListItemCard/PostListItemCard.generated';
+
+const useStyles = makeStyles(() => ({
+  postListItems: {
+    maxWidth: 600,
+  },
+}));
+
+interface PostListProps {
+  parent: PostableIdFragment;
+  posts: any;
```

Comment:
@CarsonF What would be the correct type here?

Reply:
I resolved with the latest commit.